wasm/utils/solana-extra/src/account_decoder/mod.rs

```diff
@@ -43,6 +43,7 @@ fn slice_data(data: &[u8], data_slice_config: Option<UiDataSliceConfig>) -> &[u8
 #[serde(rename_all = "camelCase")]
 pub struct UiAccount {
     pub lamports: u64,
+    pub space: u64,
```

Comment:
`space` field looks to be optional: https://github.com/anza-xyz/agave/blob/584b072b9962d1e2531564bb6cfd209746210a91/account-decoder/src/lib.rs#L48

It would also be better if the ordering of this struct was exactly the same.

Also, maybe we're able to use the upstream `solana-account-decoder` crate now? I've always wanted to get rid of this `solana-extra-wasm` crate. It looks like dependencies are managed better now, so we might finally be able to do it (not necessary for this PR).

Reply:
It's string decoding so yeah option is a good idea if we don't know exactly the layout

Reply:
Let's ride off in another PR to avoid all the huge changes in one PR